src/Controls/GlobeControls.js

```diff
@@ -593,6 +595,11 @@ class GlobeControls extends THREE.EventDispatcher {
         // Initialize dolly movement.
         dollyStart.copy(event.viewCoords);
 
+        // Store first position of mouse when dolly
+        if (!dollyOrigin) {
+            dollyOrigin = this.view.getPickingPositionFromDepth(event.viewCoords);        // mouse position
+        }
+
```

Comment:
could you use `pickedPosition` instead of `dollyOrigin`, it seems to have a duplicate use

Reply:
Yes you're right, I could use `pickedPosition`. I need something to know if I'm doing a dolly (cf next comment)